src/components/Layout/Header/SideNav.tsx

```diff
@@ -1,11 +1,15 @@
 import { chakra, useColorModeValue } from '@chakra-ui/react'
+import { useWallet } from 'hooks/useWallet/useWallet'
 
 import { SideNavContent } from './SideNavContent'
 
 export const SideNav = () => {
   const bg = useColorModeValue('white', 'gray.850')
   const borderColor = useColorModeValue('gray.100', 'gray.750')
-
+  const {
+    state: { walletInfo },
+  } = useWallet()
+  const top = walletInfo?.deviceId === 'DemoWallet' ? '7rem' : '4.5rem'
```

Comment:
@reallybeard thoughts?

Reply:
LGTM -- we may need to revisit how we do these banners in the future if for some reason we need to do more than one.